src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,41 +77,55 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={movieTitleQuery}
+              onChange={handleSearchTitleInput}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {isError
+           && (
+             <p className="help is-danger" data-cy="errorMessage">
+               Can&apos;t find a movie with such a title
+             </p>
+           )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={classNames(
+                'button is-light',
+                { 'is-loading': isLoadingMovie },
+              )}
+              disabled={movieTitleQuery === ''}
             >
               Find a movie
             </button>
           </div>
 
-          <div className="control">
-            <button
-              data-cy="addButton"
-              type="button"
-              className="button is-primary"
-            >
-              Add to the list
-            </button>
-          </div>
+          {movieTitleQuery && (
```

Comment:
we need this button if the movie was found
```suggestion
          {movie !== null && (
```